homeassistant/components/vizio/config_flow.py

```diff
@@ -165,7 +173,11 @@ def __init__(self, config_entry: ConfigEntry) -> None:
                 default=self.config_entry.options.get(
                     CONF_VOLUME_STEP, DEFAULT_VOLUME_STEP
                 ),
-            ): vol.All(vol.Coerce(int), vol.Range(min=1, max=10))
+            ): vol.All(vol.Coerce(int), vol.Range(min=1, max=10)),
+            vol.Optional(
+                CONF_TIMEOUT,
```

Comment:
Do we really need the timeout option? We try to avoid adding timeout options and just set a sane default.

Reply:
Yes that's fine, I will remove this. I can tune the sane default based on feedback.